src/ui/views/Connect/Connect.tsx

```
@@ -152,7 +159,12 @@ export const Connect = ({ node }: Props) => {
               key={key.id}
               data={key}
               onClick={() =>
-                handleConnect(key.name, key.namespace, key.translation)
+                handleConnect(
+                  key.id,
+                  key.name,
+                  key.namespace,
+                  key.translation,
+                )
```

ESLint

[error] Replace `⏎··················key.id,⏎··················key.name,⏎··················key.namespace,⏎··················key.translation,⏎················` with `key.id,·key.name,·key.namespace,·key.translation`

(prettier/prettier)